packages/fxa-content-server/app/scripts/views/pair/index.js

```diff
@@ -65,22 +65,17 @@ class PairIndexView extends FormView {
   }
 
   setInitialContext(context) {
-    // Check the entrypoint. If we aren't in a entrypoint=fxa_app_menu context,
-    // then don't ask if the user needs to download Fx mobile then show QR code.
     const graphicId = this.getGraphicsId();
-    const askMobileStatus = this.showDownloadFirefoxQrCode();
     const needsMobileConfirmed = this.model.get('needsMobileConfirmed');
 
-    if (askMobileStatus) {
-      GleanMetrics.cadFirefox.choiceView();
-    }
     if (needsMobileConfirmed) {
       GleanMetrics.cadFirefox.view();
+    } else {
+      GleanMetrics.cadFirefox.choiceView();
     }
 
     context.set({
```

Comment:
I added the header but was spending way to much time trying to get the checkmark styled correctly, will file a follow up.

Reply:
@vbudhram Sounds good, I think in that same ticket we should change this to not be a `div` instead of an `h1` especially since there's an h1/h2 right after this.